src/org/openlcb/cdi/CdiRep.java

```diff
@@ -100,7 +100,7 @@ public static interface IntegerRep extends Item {
         public boolean isSliderHint();
         // Does the slider itself immediately write its value on change?
         public boolean isSliderImmediate();
-        public int getSliderDivisions();
+        public int getSliderTickSpacing();
```

Comment:
Let's add a documentation comment that tells what this value is, especially the error case.

Reply:
Thanks, done!